src/views/users/User/UserList.vue

```diff
@@ -65,7 +67,16 @@ export default {
             }
           },
           mfa_level: {
-            width: '130px'
+            width: '130px',
+            formatter: (row) => {
+              if (store.getters.publicSettings['SECURITY_MFA_AUTH'] === MFASystemSetting.allUsers) {
+                return this.$t('MFAAllUsers')
+              } else if (store.getters.publicSettings['SECURITY_MFA_AUTH'] === MFASystemSetting.onlyAdminUsers && (row?.is_superuser || row?.is_org_admin)) {
+                return this.$t('MFAOnlyAdminUsers')
+              } else {
+                return row['mfa_level'].label
+              }
+            }
           },
           source: {
             width: '120px',
```

Comment:
The changes between the updated code and its previous version do not seem to affect the performance significantly.

In terms of issues or optimizations:

For the `getDayFuture()` function, it's good practice to make sure that it always returns a valid date object. If you're working with dates outside of current system settings, this can cause inconsistencies in your output.

It might be beneficial to add more type hints around the imported constants and functions to better document any potential errors or unexpected behavior within these points.

If importing multiple different packages across components could lead to an unclear dependency tree, consider consolidating all imports into one place at the top level. This can help maintain consistency throughout the project if needed.

Lastly, while readability is important, ensure there isn't overcomplication that obscures functionality or doesn't simplify operations where possible. For instance, the MFALevels component appears unnecessarily complex, so perhaps refactoring it down towards clarity may improve overall efficiency without sacrificing major functionality. 

Overall, keep in mind that improvements require understanding both functional requirements of the components themselves and larger context including dependencies and how they interact.

Remember to test your updates thoroughly after merging them back to maintain quality standards.